canvasapi/course.py

```diff
@@ -945,6 +945,63 @@ def create_external_tool(self, name, privacy_level, consumer_key, shared_secret,
 
         return ExternalTool(self._requester, response_json)
 
+
+    def list_external_feeds(self):
```

Comment:
Sorry, another mistake on my part. Please remove one of the blank lines above this function.